### ISSUE TITLE
libc.so.6(GLIBC_PRIVATE)(64bit)

### ISSUE THREAD

**Issue by tim77** [flightlessmango/MangoHud#75]
Hello. Can't install new 0.3.0 version in Fedora:

```
Problem 1: conflicting requests
  - nothing provides libc.so.6(GLIBC_PRIVATE)(64bit) needed by mangohud-0.3.0-1.fc32.x86_64
```

Tested on F31 and F32. Do we really need this private API's?

**Comment by jackun** [comment]
Is it the `__libc_dlsym`?

**Comment by AngryPenguinPL** [comment]
Hi.
I have exactly the same problem in OpenMandriva Cooker. Just trying to import package to system repository and build is completed without any issues but it fail on test with:

`nothing provides libc.so.6(GLIBC_PRIVATE)(64bit) needed by mangohud-0.3.0`

PS. for some reasons, we do not provide in our distribution GLIBC_PRIVATE, so this is intentional.

Anyway I can force installing it with rpm -i --nodeps (just to check if it works) but then every games segfault at launch if load it with mangohud parametr.
Not sure, if this is caused by missing GLIBC_PRIVATE or sometings else.

**Comment by jackun** [comment]
If you can try `develop` branch.

**Comment by tim77** [comment]
> If you can try `develop` branch if it is fixed there.

Tried https://github.com/flightlessmango/MangoHud/commit/c7bee1a99d4fce3943ed4fbf990638b6be460376 and no this issue!

**Comment by jackun** [comment]
Should be fixed v0.3.1 in master now.

**Comment by AngryPenguinPL** [comment]
> Should be fixed v0.3.1 in master now.

Yes, I can confirm 0.3.1 fix issue with GLIBC_PRIVATE and also segfault.

Thanks.
